src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -148,14 +150,23 @@ void eventSource_ErrorRaised(object sender, BuildErrorEventArgs e)
         }
 
 
-        public void Shutdown() {
-            // Keep open if autoscroll disabled (the user is reading info)
-            /*while (FancyLoggerBuffer.AutoScrollEnabled || !FancyLoggerBuffer.IsTerminated)
-            {
-            }*/
+        public void Shutdown()
+        {
             FancyLoggerBuffer.Terminate();
+            // TODO: Remove. There is a bug that causes switching to main buffer without deleting the contents of the alternate buffer
             Console.Clear();
-            Console.WriteLine("Build status, warnings and errors will be shown here after the build has ended and the interactive logger has closed");
+            // Console.WriteLine("Build status, warnings and errors will be shown here after the build has ended and the interactive logger has closed");
+            if (Succeeded)
+            {
+                Console.WriteLine(ANSIBuilder.Formatting.Color("Build succeeded.", ANSIBuilder.Formatting.ForegroundColor.Green));
+                Console.WriteLine("\t0 Warning(s)");
```

Comment:
Nit: A build can succeed with warnings (ours generally have warnings-as-errors turned on but that's not required).

```suggestion
                Console.WriteLine("\tX Warning(s)");
```